src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -273,5 +279,14 @@ public override ParserBase DecorateCodeParser([NotNull] ParserBase incomingCodeP
                     CreateModelExpressionMethodName = CreateModelExpressionMethod
                 });
         }
+
+        private IReadOnlyList<ChunkTree> GetInheritedChunkTrees(string sourceFileName)
+        {
+            var inheritedChunkTrees = GetInheritedChunkTreeResults(sourceFileName)
+                .Select(result => result.ChunkTree)
+                .ToList();
```

Comment:
We rely on the ordering of chunks to inject them in the page being generated. The `IList` contract is meant to indicate this ordering.